package.json

```diff
@@ -51,8 +51,8 @@
     "@babel/core": "^7.12.9",
     "@babel/preset-env": "^7.12.7",
     "babel-loader": "^8.2.2",
-    "identity-style-guide": "^4.0.0",
-    "uswds": "^2.9.0",
+    "identity-style-guide": "^6.2.0",
+    "uswds": "^2.13.0",
```

Comment:
We hopefully shouldn't need the dependency anymore. It was needed for the code removed above to `@import "uswds/dist/scss/components/identifier";`.